index.bs

```diff
@@ -6964,11 +6967,11 @@ to its caller, and so on.
 </div>
 
 
-<h3 id="es-environment">ECMAScript environment</h3>
+<h3 id="js-environment" oldids="es-environment">JavaScript environment</h3>
```

Comment:
I think keeping the old IDs and not adding new IDs would be fine as well. I personally prefer that as it shows the document has some history.

Reply:
There's some history of using `oldids` (e.g., adding `es-undefined` and adding `es-void` as an old ID, when WebIDL renamed the concept of `void`). If the consensus is that we should continue to use `es-` for IDs then I can unwind that, though.